GET_STARTED.md

```diff
@@ -0,0 +1,91 @@
+# Get started
+
+## Structure
+
+Our codebase is split into multiple repos: `website`, `api`, `app`. We run those 3 node.js applications separately. `website` and `app` are the client applications that you can find on `https://opencollective.com/yeoman` (`website`) and `https://app.opencollective.com/` (`app`).
+
+To run one of the client application locally, you need to run the `api` alongside.
+
+## Setup api
+
+### Install postgres (MacOSX)
+
+```bash
+brew install postgres
+pg_ctl -l /usr/local/var/postgres/server.log start
+```
+
+Or download [Postgres.app](http://postgresapp.com)
+
+### Install API
+
+We currently use node `5.1.0` and npm `3.3.12`. Checkout [nvm](https://github.com/creationix/nvm) for versioning node.js binaries.
+
+```bash
+git clone https://github.com/OpenCollective/api.git
+cd api
+touch .env
+npm install
+npm install -g nodemon
+```
+
+You should be now able to run the api with `npm run dev` in autoreload mode or `npm start`.
+
+### Run the api
+
+```bash
+npm run db:reset
+npm run dev
+```
+
+If you have a problem with this step, you might need to grant privileges to your database (see below).
+
+Check if the api is running properly by opening `http://localhost:3060/status` in your browser.
+
+Some environment variables might be missing from your setup, you need to add them in your `.env`. If you want to test image uploading, you will need to add your own AWS config.
+
+#### Postgres privileges
+
+Now, assuming the postgres database superuser is `postgres`, let's create the databases.
+
+```bash
+createdb -U postgres opencollective_localhost
+createuser -U postgres opencollective
+psql -U postgres
+> GRANT ALL PRIVILEGES ON DATABASE opencollective_localhost TO opencollective;
+```
+
+## Run the website
+
+You will need to install gulp to compile the assets: https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md
```

Comment:
`npm install` will automatically install it no? Maybe we need to reference `./node_modules/.bin/gulp` instead of expecting `gulp` to be available as a global command?